tests/test_python-opensubtitles.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+from os import path
+
+
+class TestPythonOpenSubtitles(object):
+    @staticmethod
+    @pytest.fixture
+    def data():
+        class Data:
+            username = 'doctest'
+            password = 'doctest'
+            name = 'Dark City'
+            path = '/home/alex/Videos/Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo/'
+            video = 'Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo.mp4'
+            subtitle = 'Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo.srt'
+        return Data
+
+    @staticmethod
+    @pytest.fixture
+    def os():
+        from pythonopensubtitles.opensubtitles import OpenSubtitles
+        return OpenSubtitles()
+
+    @staticmethod
+    @pytest.fixture
+    def file(data):
+        from pythonopensubtitles.utils import File
+        return File(path.join(data.path, data.video))
+
+    # Login
+    @staticmethod
+    def test_login(data, os):
+        token = os.login('bad@mail.com', 'badpassword')
```

Comment:
This will try to make a call, it shouldn't do real calls in a unit test. However, maybe we should just focus in integration since this is just a client for an API.

What do you think?